### ISSUE TITLE
About the code "images = torch.cat(images, dim=0)"

### ISSUE THREAD

**Issue by meihao5631** [sthalles/SimCLR#47]
hi~ the "images" here is a tensor, but the first parameter in cat() is a tensor sequence. I don't understand what's going on here. Could you please explain it, thank you very much.

**Comment by jingyi-y** [comment]
hi I have the same question. Did you figure out why?
